src/modules/fancyzones/editor/FancyZonesEditor/MainWindow.xaml.cs

```diff
@@ -242,5 +243,28 @@ private void CloseButton_Click(object sender, RoutedEventArgs e)
         {
             this.Close();
         }
+
+        private void ResetButton_Click(object sender, RoutedEventArgs e)
+        {
+            var overlay = App.Overlay;
+            MainWindowSettingsModel settings = ((App)Application.Current).MainWindowSettings;
+
+            if (overlay.CurrentDataContext is LayoutModel model)
+            {
+                model.IsSelected = false;
+                model.IsApplied = false;
+            }
+
+            overlay.CurrentLayoutSettings.ZonesetUuid = settings.BlankModel.Uuid;
+            overlay.CurrentLayoutSettings.Type = LayoutType.Blank;
+            overlay.CurrentDataContext = settings.BlankModel;
+
+            App.FancyZonesEditorIO.SerializeAppliedLayouts();
```

Comment:
This line is not needed as we serialize when Editor is closed and read from C++ only when we detect that Editor is closed. So this is just double serialization